doc/source/tools/inlet_profile_generator/inlet_profile_generator.rst

```diff
@@ -0,0 +1,49 @@
+########################
+Inlet profile generator
+########################
+
+This tool is a simple Python script that generates a polynomial function to use as inlet boundary condition for the flow.
+
+The output of the script is a polynomial function that can be used in the ``Function expression`` parameter of a ``inlet`` boundary condition for the velocity.
+
+It can be found in the ``/contrib/utilities/inlet_profile_generator`` folder of the Lethe repository.
+
+To run the script you just need to have `SymPy <https://www.sympy.org/en/index.html>`_ installed in your Python environment. You can install it via pip:
+
+.. code-block:: bash
+    :class: copy-button
+
+    pip install sympy
+
```

Comment:
I am not sure we need to provide the sympy install instruction. Especially since on ubuntu machines this does not really work like this anymore and you need to have your virtual environment not to conflict with system packages. I would remove the installation instruction and just say you just need to have Sympy and Matplotlib.